### PR TITLE
DEVOPS-3193: tune system for Mongo, reduce logs for prod

### PR DESCRIPTION
**Pull request by wanix**
https://docs.mongodb.com/manual/administration/production-checklist-operations/
https://docs.mongodb.com/manual/administration/production-notes/
 https://docs.mongodb.com/manual/reference/ulimit/#recommended-ulimit-settings

### CODE REVIEW

File: hieradata/puppet_role/mongodb.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@
 mongodb::globals::version: '2.6.12-1'
 mongodb::server::verbose: true
 # mongodb::server::verbositylevel, from 'v' to 'vvvvv'
-mongodb::server::verbositylevel: 'v'
+# mongodb::server::verbositylevel: 'v'
```
Comment (jmlrt) [inline hieradata/puppet_role/mongodb.yaml]: you could remove this line as it's already in comment just above it

File: site/profile/manifests/mongodb.pp [hunk 2]
```diff
@@ -53,6 +53,55 @@
...
+    ensure => file,
+    path   => '/etc/init.d/disable-transparent-hugepages',
+    source => 'puppet:///modules/profile/etc/init.d/disable-transparent-hugepages',
+    mode   => '0755',
+    owner  => 'root',
+    group  => 'root',
+    before => Class['::mongodb::server'],
+    notify => [
+      Exec['enable disable-transparent-hugepages'],
+      Exec['enable disable-transparent-hugepages']
```
Comment (jmlrt) [inline site/profile/manifests/mongodb.pp]: duplicate line. did you want to add `start disable-transparent-hugepages`?

Comment (jmlrt) [reply site/profile/manifests/mongodb.pp]: do we want to run the script directly at puppet runtime (while mongodb may already be running) or only after instance restart or manual start?

Comment (wanix) [reply site/profile/manifests/mongodb.pp]: you're right, typo, I'll fix it

**Pull request merged**